static/src/javascripts/bootstraps/common.js

```diff
@@ -227,6 +227,25 @@ define([
                 });
             },
 
+            // Adds a global window:throttledScroll event to mediator, which throttles
+            // scroll events until there's a spare animationFrame.
+            // Callbacks of all listeners to window:throttledScroll are run in the
+            // same animationFrame, meaning they can all perform DOM reads for free
+            // (after the first one that needs layout triggers it).
+            // However, this means it's VITAL that all writes in callbacks are delegated to fastdom
+            throttledScrollEvent: function () {
+                var running = false;
+                window.addEventListener('scroll', function () {
+                    if (!running) {
+                        running = true;
+                        requestAnimationFrame(function () {
```

Comment:
yeah polyfilled here https://github.com/guardian/frontend/blob/master/common/app/templates/headerInlineJS/polyfills.scala.js#L59.

what would we gain from using `defer()` here?